From lieb@canonical.com  Mon Aug 17 12:52:02 2009
From: Jim Lieb <lieb@canonical.com>
Date: Wed, 12 Aug 2009 14:54:12 -0700
Subject: Staging: vt665x: remove tpci.h file
To: Greg KH <greg@kroah.com>, Forest Bond <forest@alittletooquiet.net>, Bartlomiej Zolnierkiewicz <bzolnier@gmail.com>
Cc: jim.lieb@canonical.com, Jim Lieb <lieb@canonical.com>
Message-ID: <1250114056-13843-11-git-send-email-lieb@canonical.com>


Convert pci register/command defs to use kernel definitions.
Remove tpci.h.

Signed-off-by: Jim Lieb <lieb@canonical.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---
 drivers/staging/vt6655/device.h      |    1 
 drivers/staging/vt6655/device_main.c |   12 +--
 drivers/staging/vt6655/tpci.h        |  113 -----------------------------------
 3 files changed, 6 insertions(+), 120 deletions(-)
 delete mode 100644 drivers/staging/vt6655/tpci.h

--- a/drivers/staging/vt6655/device.h
+++ b/drivers/staging/vt6655/device.h
@@ -95,7 +95,6 @@
 #include "mib.h"
 #include "srom.h"
 #include "rc4.h"
-#include "tpci.h"
 #include "desc.h"
 #include "key.h"
 #include "mac.h"
--- a/drivers/staging/vt6655/device_main.c
+++ b/drivers/staging/vt6655/device_main.c
@@ -1180,17 +1180,17 @@ static BOOL device_get_pci_info(PSDevice
 
     pDevice->pcid = pcid;
 
-    pci_read_config_byte(pcid, PCI_REG_COMMAND, &b);
-    pci_write_config_byte(pcid, PCI_REG_COMMAND, (b|COMMAND_BUSM));
+    pci_read_config_byte(pcid, PCI_COMMAND, &b);
+    pci_write_config_byte(pcid, PCI_COMMAND, (b|PCI_COMMAND_MASTER));
 
 #ifdef	PLICE_DEBUG
-   	//pci_read_config_word(pcid,PCI_REG_MAX_LAT,&max_lat);
+   	//pci_read_config_word(pcid,PCI_MAX_LAT,&max_lat);
 	//printk("max lat is %x,SubSystemID is %x\n",max_lat,pDevice->SubSystemID);
 	//for (ii=0;ii<0xFF;ii++)
-	//pci_read_config_word(pcid,PCI_REG_MAX_LAT,&max_lat);
+	//pci_read_config_word(pcid,PCI_MAX_LAT,&max_lat);
 	//max_lat  = 0x20;
-	//pci_write_config_word(pcid,PCI_REG_MAX_LAT,max_lat);
-	//pci_read_config_word(pcid,PCI_REG_MAX_LAT,&max_lat);
+	//pci_write_config_word(pcid,PCI_MAX_LAT,max_lat);
+	//pci_read_config_word(pcid,PCI_MAX_LAT,&max_lat);
 	//printk("max lat is %x\n",max_lat);
 
 	for (ii=0;ii<0xFF;ii++)
--- a/drivers/staging/vt6655/tpci.h
+++ /dev/null
@@ -1,113 +0,0 @@
-/*
- * Copyright (c) 1996, 2003 VIA Networking Technologies, Inc.
- * All rights reserved.
- *
-
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
- * the Free Software Foundation; either version 2 of the License, or
- * (at your option) any later version.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- *
- * You should have received a copy of the GNU General Public License along
- * with this program; if not, write to the Free Software Foundation, Inc.,
- * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
-
- *
- *
- * File: tpci.h
- *
- * Purpose: PCI routines
- *
- * Author: Tevin Chen
- *
- * Date: May 21, 1996
- *
- */
-
-#ifndef __TPCI_H__
-#define __TPCI_H__
-
-#include "ttype.h"
-
-/*---------------------  Export Definitions -------------------------*/
-#define MAX_PCI_BUS             4       // max. # of PCI bus that we support
-#define MAX_PCI_DEVICE          32      // max. # of PCI devices
-
-
-//
-// Registers in the PCI configuration space
-//
-#define PCI_REG_VENDOR_ID       0x00    //
-#define PCI_REG_DEVICE_ID       0x02    //
-#define PCI_REG_COMMAND         0x04    //
-#define PCI_REG_STATUS          0x06    //
-#define PCI_REG_REV_ID          0x08    //
-#define PCI_REG_CLASS_CODE      0x09    //
-#define PCI_REG_CACHELINE_SIZE  0x0C    //
-#define PCI_REG_LAT_TIMER       0x0D    //
-#define PCI_REG_HDR_TYPE        0x0E    //
-#define PCI_REG_BIST            0x0F    //
-
-#define PCI_REG_BAR0            0x10    //
-#define PCI_REG_BAR1            0x14    //
-#define PCI_REG_BAR2            0x18    //
-#define PCI_REG_CARDBUS_CIS_PTR 0x28    //
-
-#define PCI_REG_SUB_VEN_ID      0x2C    //
-#define PCI_REG_SUB_SYS_ID      0x2E    //
-#define PCI_REG_EXP_ROM_BAR     0x30    //
-#define PCI_REG_CAP             0x34    //
-
-#define PCI_REG_INT_LINE        0x3C    //
-#define PCI_REG_INT_PIN         0x3D    //
-#define PCI_REG_MIN_GNT         0x3E    //
-#define PCI_REG_MAX_LAT         0x3F    //
-
-#define PCI_REG_MAX_SIZE        0x100   // maximun total PCI registers
-
-
-//
-// Bits in the COMMAND register
-//
-#define COMMAND_BUSM        0x04        //
-#define COMMAND_WAIT        0x80        //
-
-/*---------------------  Export Types  ------------------------------*/
-
-/*---------------------  Export Macros ------------------------------*/
-
-// macro MAKE Bus Dev Fun ID into WORD
-#define MAKE_BDF_TO_WORD(byBusId, byDevId, byFunId) \
-    MAKEWORD(                                       \
-        ((((BYTE)(byDevId)) & 0x1F) << 3) +         \
-            (((BYTE)(byFunId)) & 0x07),             \
-        (byBusId)                                   \
-        )
-
-#define GET_BUSID(wBusDevFunId) \
-    HIBYTE(wBusDevFunId)
-
-#define GET_DEVID(wBusDevFunId) \
-    (LOBYTE(wBusDevFunId) >> 3)
-
-#define GET_FUNID(wBusDevFunId) \
-    (LOBYTE(wBusDevFunId) & 0x07)
-
-
-/*---------------------  Export Classes  ----------------------------*/
-
-/*---------------------  Export Variables  --------------------------*/
-
-/*---------------------  Export Functions  --------------------------*/
-
-
-
-
-#endif // __TPCI_H__
-
-
